<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>camtrak</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gview">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>871</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="capture_btn">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Known World Points</string>
    </property>
    <widget class="QTableWidget" name="known_geo_points_tbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Intrinsic Parameters</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lat</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lon</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Alt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IFOV</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>cx</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>cy</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Phi</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Theta</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Psi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="camera_lat_line">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>178.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>-158.849411000000003</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="camera_lon_line">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>21.304611999999999</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="camera_alt_line"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="camera_ifov_line">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="cx_line"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="cy_line"/>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="phi_line"/>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="theta_line"/>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="psi_line"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Distortion Coeeficients</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>k1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>k2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>k3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>p1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>p2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="k1_line">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="k2_line">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="k3_line">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="p1_line">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="p2_line">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="solve_pnp_btn">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve PnP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrate_btn">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1244</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_image"/>
    <addaction name="action_open_camera"/>
    <addaction name="action_save_parameters"/>
    <addaction name="action_load_parameters"/>
    <addaction name="action_save_fits"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionMetadata"/>
    <addaction name="actionCalibrate"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="action_calibrate"/>
    <addaction name="action_pnp"/>
    <addaction name="action_track"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMetadata">
   <property name="text">
    <string>Metadata</string>
   </property>
  </action>
  <action name="action_save_parameters">
   <property name="text">
    <string>Save Parameters</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="action_calibrate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="action_pnp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PnP</string>
   </property>
  </action>
  <action name="action_track">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
  </action>
  <action name="action_open_image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="action_open_camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="action_save_fits">
   <property name="text">
    <string>Save FITS</string>
   </property>
  </action>
  <action name="action_load_parameters">
   <property name="text">
    <string>Load Parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
